packages/cli/src/index.js

```diff
@@ -118,3 +94,16 @@ prog.parse(process.argv, {
 		);
 	},
 });
+
+function exec(cmd) {
+	cmd.catch(catchExceptions);
+}
```

Comment:
Making an honest attempt to catch errors, rather than relying entirely on `process.on('unhandledRejection')` to catch everything.